schemas/event.yml

```diff
@@ -38,53 +38,174 @@
       example: 4648
 
     - name: kind
-      level: extended
+      level: core
       type: keyword
-      short: The kind of the event.
+      short: The kind of the event. (Categorization Field)
       description: >
-        The kind of the event.
-
-        This gives information about what type of information the event
-        contains, without being specific to the contents of the event.
-      example: event
-      accepted_values:
-      - name: alert
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-      - name: event
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
+        This is one of four ECS Categorization Fields, and indicates the
+        highest level in the ECS category hierarchy.
+
+        `event.kind` gives high-level information about what type of information the
+        event contains, without being specific to the contents of the event.
+        For example, values of this field distinguish alert events from
+        metric events.
+      example: alert
+      allowed_values:
+        - name: alert
+          description: >
+            `event.kind:alert` indicates an event that describes an alert.
+            Alerts are often associated with detection rules. `event.kind:alert`
+            is often populated for events coming from firewalls, intrusion detection
+            systems, endpoint detection and response systems, etc.
+            It is used to indicate that an alert was triggered.
+        - name: event
+          description: >
+            `event.kind:event` is the most general and most common value of this
+            field. It is used to represent events that indicate that something happened.
+        - name: metric
+          description: >
+            Used to indicate that this event is a measurement taken at given point in time.
```

Comment:
So, I'm not too huge of a fan of the inconsistency in the sentence syntax here--`alert` and `event` values both start off with `event.kind:*` while the others just start with a predicate or a reference back to the field value (i.e. 'This value')--we should be consistent--visually I'm not a fan either.

If we want to follow the format of the top-level docs for the fields, it makes sense to me to start everything with either "This value" or with `event.kind:*`.

Reply:
I agree there's an inconsistency there. I think starting each definition with "This value" reads a little more easily.

However whenever need to expand a little more or give additional examples, going with the more explicit `fieldname:value` is much easier.

Perhaps we can always start with "This value" and always use `fieldname:value` at any other time where we need to mention a field and a value later?

I don't have a strong opinion on this either.

Reply:
+1 for Mat's suggestion - start each with with "This value..."   Use `fieldname:value` for subsequent discussion.